internal/worktree/sync_build_output_test.go

```
@@ -0,0 +1,250 @@
+			t.Fatal(err)
+		}
+		assertMergedLanePreserved(t, fixture, "topic/symlink-bin", headOID)
+		if got, err := os.Readlink(filepath.Join(path, "bin")); err != nil || got != target {
+			t.Fatalf("ignored bin symlink changed: target=%q err=%v", got, err)
+		}
+	})
+
+	t.Run("tracked change beneath bin", func(t *testing.T) {
+		fixture := newGitFixture(t)
+		configureIgnoredBuildOutput(t, fixture, "bin/\n")
+		path, headOID := createPublishedLane(t, fixture, "topic/tracked-bin")
```

golangci-lint (2.12.2)

[error] ineffectual assignment to headOID

(ineffassign)